cmd/storj-sim/network.go

```diff
@@ -205,6 +211,21 @@ func newNetwork(flags *Flags) (*Processes, error) {
 			}
 
 			// TODO: maybe all the config flags should be exposed for all processes?
+			apiKey := vip.GetString("client.api-key")
+			if apiKey == "" {
+				parts := strings.Split(satellite.Address, ":")
```

Comment:
Use https://golang.org/pkg/net/#SplitHostPort, using `strings.Split` doesn't work for IPv6 properly...